src/tt_citites_skylines/core.rb

```diff
@@ -11,43 +11,32 @@ module TT::Plugins::CitiesSkylinesTools
 
 
   require "tt_citites_skylines/exporter.rb"
+  require "tt_citites_skylines/exporter_dae.rb"
   require "tt_citites_skylines/guide_grid.rb"
 
 
   unless file_loaded?(__FILE__)
+    menu = UI.menu("Plugins").add_submenu(PLUGIN_NAME)
+    toolbar = UI::Toolbar.new(PLUGIN_NAME)
 
     # Commands
-
-    cmd = UI::Command.new("Export Asset") {
-      self.export_asset
+    toolbaritems = ["Export CSL FBX", "Export CSL DAE", "Guide Grid Config", "Increase Subdivision Level", "Decrease Subdivision Level", "One Floor Up", "One Floor Down" ]
```

Comment:
yeah wasn't sure if I'm able to use multi dimensional arrays. imho its definitely better then before copying the same stuff over and over again?